Sprint-2/implement/querystring.js

```diff
@@ -6,8 +6,9 @@ function parseQueryString(queryString) {
   const keyValuePairs = queryString.split("&");
 
   for (const pair of keyValuePairs) {
-    const [key, value] = pair.split("=");
-    queryParams[key] = value;
+    const [key, ...rest] = pair.split("=");
```

Comment:
This looks good, it is more complex than it needs to be though. It is removing some characters and then adding them again. Can you think of a simpler way?